lib.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+_is_automated_event() {
+	AUTOFIX_EVENTS=${AUTOFIX_EVENTS:-push}
+
+	if echo "${GITHUB_EVENT_NAME}" | grep -Eq "^$AUTOFIX_EVENTS$"; then
```

Comment:
Maybe change it to bash's pattern match:

```suggestion
	if [[ "${GITHUB_EVENT_NAME}" =~ ^($AUTOFIX_EVENTS)$; then
```

Where `AUTOFIX_EVENTS` have event names separated by `|`.